src/earn/EarnEnterAmount.tsx

```diff
@@ -336,27 +402,32 @@ function EarnEnterAmount({ route }: Props) {
                 borderRadius={TOKEN_SELECTOR_BORDER_RADIUS}
                 onPress={onTokenPickerSelect}
                 style={styles.tokenSelectButton}
-                disabled={tokens.length === 1}
+                disabled={!dropdownEnabled}
                 testID="EarnEnterAmount/TokenSelect"
               >
                 <>
-                  <TokenIcon token={token} size={IconSize.SMALL} />
-                  <Text style={styles.tokenName}>{token.symbol}</Text>
-                  {tokens.length > 1 && <DownArrowIcon color={Colors.gray5} />}
+                  <TokenIcon
+                    token={isWithdrawal ? depositTokens[0] : inputToken}
+                    size={IconSize.SMALL}
+                  />
+                  <Text style={styles.tokenName}>
+                    {isWithdrawal ? depositTokens[0].symbol : inputToken.symbol}
```

Comment:
same here re `inputToken`

Reply:
Changed in 1da7fac29afd55d61d0717cdc66e28568f4cdca2.